pages/UI.php

```diff
@@ -1315,6 +1317,11 @@ function DisplayNavigatorGroupTab($oP)
 								$oObj = MetaModel::GetObject(get_class($oObj), $oObj->GetKey());
 								$oObj->UpdateObjectFromPostedForm('', array_keys($aExpectedAttributes), $aExpectedAttributes);
 								$sIssues = $e->getMessage();
+								
+								if($e instanceof CoreCannotSaveObjectException) {
+									$oException = $e;
+								}
```

Comment:
What if you would do this instead? Then there's no need to do the if/else later on.
```suggestion
								$sIssues = $e->getHtmlMessage();
```
(correct styling would still something to be done in CSS, maybe separate PR)

Reply:
I'm not sure whether or not we can expect other exceptions here, which don't support the `getHtmlMessage()`, so I was being cautious :)

Reply:
Right, `Coreexception` doesn’t seem to have this method 🤔

Reply:
Hey, we'll work on creating this method in CoreException, so this fix will be more generic.